simtools/schemas/metadata.schema.yml

```diff
@@ -0,0 +1,511 @@
+          links to the data model definition.
+        required:
+          - DATA
+          - DESCRIPTION
+          - FORMAT
+          - ID
+          - VALID
+        type: object
+        additionalProperties: false
+        properties:
+          DESCRIPTION:
+            anyOf:
+              - type: string
+              - type: "null"
+            default: null
+            description: Product description
+          CREATION_TIME:
+            anyOf:
+              - type: string
+                format: date-time
+              - type: "null"
+            format: date-time
+            default: null
+            description: |-
+              Human-readable date and time of file creation, in ISO format.
+          ID:
+            anyOf:
+              - type: string
+                format: uuid
+              - type: "null"
+              - type: number
+            default: null
+            description: |-
+              Product identifier.
+          DATA:
+            title: Data
+            required:
+              - CATEGORY
+              - ASSOCIATION
+              - LEVEL
+              - MODEL
+            type: object
+            additionalProperties: false
+            properties:
+              CATEGORY:
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+                description: |-
+                  CTA data identifier
+                enum:
+                  - SIM
+                  - CAL
+                  - null
+              ASSOCIATION:
+                anyOf:
+                  - type: "null"
+                  - type: string
+                default: null
+                description: |-
+                  Array element association.
+                enum:
+                  - CTA
+                  - Site
+                  - Subarray
+                  - Telescope
+                  - Target
+                  - null
+              LEVEL:
+                type: string
+                default: R1
+                description: |-
+                  CTA data level
+                enum:
+                  - R0
+                  - R1
+                  - DL0
+                  - DL1
+                  - DL2
+                  - DL3
+                  - DL4
+                  - DL5
+                  - DL6
+              TYPE:
+                type: string
+                default: Service
+                description: |-
+                  CTA data type.
+                enum:
+                  - Event
+                  - Monitoring
+                  - Service
+                  - DataCube
+                  - Catalog
+              MODEL:
+                title: Model
+                required:
+                  - NAME
+                  - VERSION
+                type: object
+                additionalProperties: false
+                properties:
+                  NAME:
```

Comment:
I've seen there are some differences when comparing the table in the pdf with the implementation here. For example, in the pdf we only use one level of subitem, but here there are multiple (e.g.Product/model/name, while in the pdf document we have Product/data model name. I'm commenting this because I'm not sure whether we need to be that faithful to the pdf table?

Reply:
Did you check A.2.3 in the document? There are several hierarchies. Or do you mean something else?

Reply:
Ok! I was looking in the table (figure A.2). That is fine then.